File: Soumadip/string/Substring.java
package Soumadip.string;

public class Substring {
      public static void subString(String str, int val1, int val2) {
            for (int i = val1; i < val2; i++) {
                  System.out.print(str.charAt(i));
            }
      }

      public static void main(String[] args) {
            String str = "Hello World";
            String str2 = new String(str.substring(2, 7));
            System.out.println("The Substring is '" + str2 + "'");
            subString(str, 2, 7);

      }

}

File: cognizent_GENCNEXT/Substring.cs
using System;

namespace cognizent_GENCNEXT
{
    public class Substring
    {
        public static void subString(string str, int val1, int val2)
        {
            for (int i = val1; i < val2; i++)
            {
                Console.Write(str[i]);
            }
        }

        public static void Main(string[] args)
        {
            string str = "Hello World";
            string str2 = str.Substring(2, 5);
            Console.WriteLine("The Substring is '" + str2 + "'");
            subString(str, 2, 7);

        }

    }
}
